api/v1/updateservice_types.go

```diff
@@ -64,7 +64,6 @@ const (
 )
 
 // +kubebuilder:object:root=true
-
```

Comment:
https://github.com/kubernetes-sigs/kubebuilder/issues/1558
Otherwise, we would get an error like ^^^.